lib/mayaUsd/ufe/UsdTransform3dMayaXformStack.cpp

```diff
@@ -255,34 +170,122 @@ Ufe::Transform3d::Ptr createTransform3d(
 class UsdTRSUndoableCmdBase : public Ufe::SetVector3dUndoableCommand
 {
 private:
-    UsdGeomXformOp    fOp;
-    const UsdTimeCode fReadTime;
-    const UsdTimeCode fWriteTime;
-    const VtValue     fPrevOpValue;
-    const TfToken     fAttrName;
+    const UsdTimeCode _readTime;
+    const UsdTimeCode _writeTime;
+    VtValue           _prevOpValue;
+    VtValue           _newOpValue;
+    TfToken           _attrName;
+    UsdGeomXformOp    _op;
+    OpFunc            _opFunc;
 
 public:
+    struct State
```

Comment:
This needs much more documentation and preferably be taken out into its own class.